api/src/pcapi/routes/backoffice/finance/finance_incidents_blueprint.py

```diff
@@ -63,6 +63,12 @@ def _get_incidents(
         .outerjoin(educational_models.CollectiveBooking.collectiveStock)
     )
 
+    if form.is_collective.data and len(form.is_collective.data) == 1:
+        if form.is_collective.data[0] == "true":
+            query = query.filter(finance_models.FinanceIncident.relates_to_collective_bookings == True)
+        else:
+            query = query.filter(finance_models.FinanceIncident.relates_to_collective_bookings != True)
```

Comment:
Détail : suivant nos "bonnes pratiques" : 
```suggestion
            query = query.filter(finance_models.FinanceIncident.relates_to_collective_bookings)
        else:
            query = query.filter(sa.not_(finance_models.FinanceIncident.relates_to_collective_bookings))
```